get-started/codelab/index.md

```diff
@@ -0,0 +1,1301 @@
+            [[highlight]]},[[/highlight]]
+          [[highlight]]);[[/highlight]]
+          [[highlight]]final divided = ListTile[[/highlight]]
+              [[highlight]].divideTiles([[/highlight]]
+            [[highlight]]context: context,[[/highlight]]
+            [[highlight]]tiles: tiles,[[/highlight]]
+          [[highlight]])[[/highlight]]
+              [[highlight]].toList();[[/highlight]]
+    );
+  }
+{% endprettify %}
+</li>
+
+<li markdown="1"> The builder property returns a Scaffold.
+    From the Material library, this widget provides an app bar for the
+    new route with the name, "Saved Suggestions". The body consists of a
+    ListView with the ListTiles.
+
+Add the highlisted code below:
+
+<!-- skip -->
+{% prettify dart %}
+  void _pushSaved() {
+    Navigator.of(context).push(
+      new MaterialPageRoute(
+        builder: (context) {
+          final tiles = _saved.map(
+                (pair) {
+              return new ListTile(
+                title: new Text(
+                  pair.asPascalCase,
+                  style: _biggerFont,
+                ),
+              );
+            },
+          );
+          final divided = ListTile
+              .divideTiles(
+            context: context,
+            tiles: tiles,
+          )
+              .toList();
+
+          [[highlight]]return new Scaffold([[/highlight]]
+            [[highlight]]appBar: new AppBar([[/highlight]]
+              [[highlight]]title: new Text('Saved suggestions'),[[/highlight]]
+            [[highlight]]),[[/highlight]]
+            [[highlight]]body: new ListView(children: divided),[[/highlight]]
+          [[highlight]]);[[/highlight]]
+        },
+      ),
+    );
+  }
+
+{% endprettify %}
+</li>
+
+<li markdown="1"> Hot reload the app. Favorite some of the selections and
+    tap the list icon in the app bar. The new route appears containing
+    the favorites. Note that the Navigator adds a "Back" button to the
+    app bar. You didn't have to explicitly implement Navigator.pop.
+    Tap the back button to return to the home route.
+</li>
+
+<li markdown="1"> You can easily change an app's theme by configuring
+    the ThemeData class.  Our app uses the default Theme, but you can use
+    themes to make the app reflect your branding.
+
+You'll set the primary color of your theme to be white, but you can play
+with this feature to test other themes.
+
+Add the following code to MyApp:
+
+<!-- skip -->
+{% prettify dart %}
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Startup Name Generator',
+      [[highlight]]theme: new ThemeData([[/highlight]]
+        [[highlight]]primaryColor: Colors.white,[[/highlight]]
+      [[highlight]]),[[/highlight]]
+      home: new RandomWords(),
+    );
+  }
+}
+{% endprettify %}
+</li>
+
+<li markdown="1"> Hot reload the app. You'll notice that the background is white,
+    even the app bar.
```

Comment:
This might be a good time to suggest they play around with hot reload. Something like "Try changing the color again (Colors has a lot of constants!) and saving the code to trigger a hot reload. You can quickly experiment with a lot of UI details this way."

Reply:
I added it as an exercise for the reader. Thanks so much!